config/services.php

```diff
@@ -17,6 +17,7 @@
 
 return function (ContainerConfigurator $configurator) {
     $configurator->parameters()
+        ->set('app.version', '1.1.0')
```

Comment:
Each time we deploy Kanbanbot, we publish a new Release on GitHub (this is how the CD works).

So we need to write somewhere a "How to deploy Kanbanbot" and insert there "DONT FORGET TO UPDATE THE VERSION NUMBER IN config/services.php"

Else we'll have a different version number on github and on the healthcheck 😅